cli/src/commands/jobs.rs

```diff
@@ -241,3 +240,25 @@ pub async fn handle_submission(
     }
     Ok(CommandValue::Action(action))
 }
+
+/// Get the current project's UUID.
+///
+/// Assumes that the clap `matches` has a `project` arguments option.
+async fn project_uuid(api: &mut PhylumApi, matches: &clap::ArgMatches) -> Result<Uuid> {
+    // Prefer `--project` if it was specified.
+    if let Some(project_name) = matches.value_of("project") {
+        let response = api.get_project_details(project_name).await;
+        let project_id = response.context("Project details request failure")?.id;
+        return Uuid::parse_str(&project_id).context("Invalid project UUID");
+    }
+
+    // Retrieve the project from the `.phylum_project` file.
+    get_current_project()
+        .map(|p: ProjectConfig| p.id)
+        .ok_or_else( ||
+            anyhow!(
+                "Failed to find a valid project configuration. Specify project using `--project` flag \
+                or create on with `phylum projects create <name>`"
```

Comment:
Typo: `on` should be `one`

```suggestion
                or create one with `phylum projects create <name>`"
```